rest_framework_filters/filterset.py

```diff
@@ -70,6 +70,17 @@ def related_filters(self):
             ])
         return self._related_filters
 
+    @property
+    def expanded_filters(self):
+        if '_expanded_filters' not in self.__dict__:
```

Comment:
I wanted to be consistent with the current code so I used the intuition in `related_filters` to check `__dict__` instead of using `hasattr`. Note that my PR is to branch 0.11 and not master

Reply:
Gotcha. I didn't realize this was against the 0.11 branch. That makes sense, given the weird caching behavior. On the current master, this should no longer be an issue.